x-pack/plugins/lens/public/metric_visualization/expression.scss

```diff
@@ -11,11 +11,81 @@
     font-size: $euiSizeXXL * 2;
     font-weight: $euiFontWeightSemiBold;
     border-radius: $euiBorderRadius;
-    padding: 0 $euiSize;
   }
 
   .lnsMetricExpression__title {
     font-size: $euiSizeXL;
     color: $euiTextColor;
+    &.reversOrder {
+      order: 1;
+    }
   }
-}
\ No newline at end of file
+
+  .lnsMetricExpression_title_container {
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    &.rowDirection {
+      flex-direction: row;
+    }
+    &.alignStart {
+      align-items: start;
+    }
+    &.alignEnd {
+      align-items: end;
+    }
+    &.alignCenter {
+      align-items: center;
+    }
+    &.titleSizeXS {
+      .lnsMetricExpression__title {
+        font-size: $euiSizeXS;
+      }
+      .lnsMetricExpression__value {
+        font-size: $euiSizeS * 2;
+      }
+    }
+    &.titleSizeS {
+      .lnsMetricExpression__title {
+        font-size: $euiSizeS * 1.5;
+        font-weight: 700;
+      }
+      .lnsMetricExpression__value {
+        font-size: $euiSizeM * 2.25;
+      }
+    }
+    &.titleSizeM {
+      .lnsMetricExpression__title {
+        font-size: $euiSizeM * 1.5;
+      }
+      .lnsMetricExpression__value {
+        font-size: $euiSizeL * 2;
+      }
+    }
+    &.titleSizeL {
+      .lnsMetricExpression__title {
+        font-size: $euiSizeL;
+      }
+      .lnsMetricExpression__value {
+        font-size: $euiSizeXL * 2;
+      }
+    }
+    &.titleSizeXL {
+      .lnsMetricExpression__title {
+        font-size: $euiSizeXL;
+      }
+      .lnsMetricExpression__value {
+        font-size: $euiSizeXXL * 2;
+      }
+    }
+
+    &.titleSizeXXL {
+      .lnsMetricExpression__title {
+        font-size: $euiSizeXXL;
+      }
+      .lnsMetricExpression__value {
+        font-size: $euiSizeXXL * 3;
+      }
+    }
```

Comment:
Rather than using the `$euiSize` series of variables, would it be better to use the `$euiFontSize` series of variables?